### PR TITLE
Fix readme for env setup

### PR DESCRIPTION
**Pull request by blagoev**
updated readme for building on windows

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,12 +75,24 @@ npm install --build-from-source=realm
...
 
         - Check [node-gyp](https://github.com/nodejs/node-gyp) manual for custom installation procedure for Windows
+        
+   Install openssl libraries vith vcpkg
+  
+        ```
+        git clone https://github.com/Microsoft/vcpkg
+        cd vcpkg
+        bootstrap-vcpkg.bat
+        vcpkg install openssl:x64-windows
```
Comment (fealebenpae) [inline README.md]:
```suggestion
        vcpkg install openssl:x64-windows-static
```

File: README.md [hunk 2]
```diff
@@ -75,12 +75,24 @@ npm install --build-from-source=realm
...
+        mkdir C:\src\vcpkg\installed\x64-windows-static\lib
```
Comment (fealebenpae) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -75,12 +75,24 @@ npm install --build-from-source=realm
...
+        copy .\packages\openssl-windows_x64-windows\lib\*.lib C:\src\vcpkg\installed\x64-windows-static\lib
```
Comment (fealebenpae) [inline README.md]:
```suggestion
```

**Comment by blagoev** [comment]
@bmunkholm Resolved the homebrew explanations here.

**Pull request merged**